### PR TITLE
awsebcli: 3.12.2 -> 3.12.3

### PR DESCRIPTION
**Pull request by eqyiel**
###### Motivation for this change

New release

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/virtualization/awsebcli/default.nix [hunk 1]
```diff
@@ -67,11 +67,11 @@ let
 in with localPython.pkgs; buildPythonApplication rec {
   name = "${pname}-${version}";
```
Comment (dotlambda) [inline pkgs/tools/virtualization/awsebcli/default.nix]: Please get rid of the `name`, it isn't needed anymore.

**Review by dotlambda — APPROVED** [review]
Two things that were not perfect even before this change:
- You could use `oldAttrs.src.override`.
- The description shouldn't contain a period

**Comment by adisbladis** [comment]
@GrahamcOfBorg build awsebcli

**Comment by adisbladis** [comment]
Thanks!

**Pull request merged**